src/app/code/community/Shopgate/Cloudapi/Helper/Request.php

```diff
@@ -42,6 +42,10 @@ class Shopgate_Cloudapi_Helper_Request extends Mage_Core_Helper_Abstract
      */
     const KEY_SGCLOUD_CALLBACK_DATA = 'sgcloud_callback_data';
 
+    /**
+     * Parameter indicating a shopgate cloud is guest checkout
+     */
+    const KEY_SGCLOUD_GUEST_CHECKOUT = 'is_guest_checkout';
```

Comment:
We should maybe start with getting rid of the TD. Keys should be KEY_SGCONNECT_...

Reply:
Shopgate Cloud was just the old name of the platform and it was renamed to Shopgate Connect. All new plugins should adapt to that in any case. For this plugin and the SDK I agree and suggest to consider it as TD. You can raise that topic talking about TD with Daniel next time. For new code it should not introduce new cloud occurrences anymore, and I also suggest to refactor all occurrences around step by step.

Reply:
I remember when this question was raised when this plugin was in its infancy (and could be changed), the definitive answer came from higher ups that it will not change. It was the time where it was much easier to do, but now, not so much. So if we had not done it then, I am not sure why we are raising the issue now.